filemgr/src/main/java/org/apache/oodt/cas/filemgr/catalog/LuceneCatalog.java

```diff
@@ -546,7 +556,7 @@ private Product getProductByName(String productName, boolean getRefs)
                 } else {
                     LOG.log(Level.FINEST,
                         "Request for products returned no results");
-                    return null;
+                    return products;
```

Comment:
We can remove this line right? Since we are returning `products` at the end of this method anyway.